cmd/vcfAncestorAnnotation/vcfAncestorAnnotation.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"github.com/vertgenlab/gonomics/fasta"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/vcf"
+	"flag"
+	"fmt"
+	"log"
+)
+
+func vcfAncestorAnnotation(inFile string, faFile string, outFile string) {
+	ch, header := vcf.GoReadToChan(inFile)
+	records := fasta.Read(faFile)
+	out := fileio.EasyCreate(outFile)
+	defer out.Close()
+
+	vcf.NewWriteHeader(out, header)
+
+	for v := range ch {
+		vcf.VcfAnnotateAncestorFromFa(v, records)
+		vcf.WriteVcf(out, v)
+	}
+}
+
+func usage() {
+	fmt.Print(
+		"vcfAncestorAnnotation - Adds ancestral allele to the INFO column of entries in a VCF file.\n" +
+		"The VCF file must be split by chromosome beforehand, as the fasta file is formatted with the first entry as the reference sequence\n" +
+		"and the second entry as the inferred ancestor sequence for a particular chromosome.\n" +
+			"Usage:\n" +
+			"vcfAncestorAnnotation input.vcf records.fa output.vcf\n" +
```

Comment:
Is records.fa an alignment, or just fasta sequences?  Maybe we can make that more clear?  I think I have seen ".mfa" used for alignments.  I could also have the name "alignment.fa" in the help message to make that more clear.